.gitignore

```diff
@@ -67,8 +67,6 @@ doxygen-*
 .coverage*
 *.tmp
 *.dat
-.vscode/ipch
-.vscode/tags
-.vscode/.ropeproject
 Nuitka-*
 nuitka/tools/podman/containers/requirements-devel.txt
+.vscode/
```

Comment:
I want to know about files there, that are to be ignored, so as to not missing things not to ignore, this counteracts it, so I won't merge this part.

Reply:
so you'd rather gitignore specific files?

Reply:
Yes, already doing that, I think you removed some, those ones are vscode state files, but every now and then, an extension creates a config file, and that I would hate to accidentally ignore. So whatever brought you to ignoring that, it's worth considering if that file is for git to contain or not on a merits base.